core/src/main/java/tech/pegasys/web3signer/core/metrics/SlashingProtectionMetrics.java

```diff
@@ -34,6 +36,12 @@ public SlashingProtectionMetrics(final MetricsSystem metricsSystem) {
             ETH2_SLASHING_PROTECTION,
             "prevented_signings",
             "The number of prevented signings due to violation of slashing conditions.");
+
+    this.databaseTimer =
+        metricsSystem.createTimer(
+            ETH2_SLASHING_PROTECTION,
+            "slashing_protection_database_duration",
```

Comment:
slashing_protection prefix can be removed. The metrics name is appended with the metric category so will already have a "eth2_slashingprotection" prefix